java/memory/memory-core/src/test/java/org/apache/arrow/memory/TestBoundaryChecking.java

```diff
@@ -55,15 +55,15 @@ private boolean getFlagValue(ClassLoader classLoader) throws Exception {
   }
 
   /**
-   * Ensure the flag for bounds checking is enabled by default.
+   * Siren: Ensure the flag for bounds checking is disabled by default.
    * This will protect users from JVM crashes.
```

Comment:
If we disable this by default, this line can be removed or negated

Reply:
For example:
```suggestion
   * Enabling it will protect users from JVM crashes.
```

Reply:
Thanks, done https://github.com/sirensolutions/arrow/pull/35/commits/cfd965270c0af9dda02e963f5e5711988d4633d9